apps/studio/src/components/Searchbar/SearchModalBodyContentStates.tsx

```diff
@@ -70,39 +91,67 @@ export const InitialState = ({
   siteId: string
   items: SearchResultResource[]
 }) => {
+  const { get } = useResourceLocalViewHistory({ siteId })
+  const { data: localViewHistorySearchResults = [] } =
+    trpc.resource.searchWithResourceIds.useQuery({
+      siteId,
+      resourceIds: get().map((history) => history.resourceId),
```

Comment:
if a resource was deleted, will it still return a result

Reply:
nope, because the underlying query is using a `IN`

https://github.com/opengovsg/isomer/blob/e64857553ada4ff98601d0fee13ef7445e0b065f/apps/studio/src/server/modules/resource/resource.service.ts#L607-L617